website/content/docs/connect/dataplane/consul-dataplane.mdx

```diff
@@ -0,0 +1,148 @@
+---
+layout: docs
+page_title: Consul Dataplane CLI Reference
+description: >-
+   Consul Dataplane runs as a separate binary controlled with the `consul-dataplane` CLI command. Learn how to use this command to configure your dataplane on Kubernetes with this reference guide and example code.
+---
+
+# Consul Dataplane CLI Reference
+
+The `consul-dataplane` command interacts with the binary for [simplified service mesh with Consul Dataplane](/consul/docs/k8s/dataplane/index). Use this command to install Consul Dataplane, configure its Envoy proxies, and secure Dataplane deployments.
+
+## Usage
+
+Usage: `consul-dataplane [options]`
+
+### Requirements
+
+Consul Dataplane requires servers running Consul version `v1.14-beta+`. To find a specific version of Consul, refer to [Hashicorp's Official Release Channels](https://www.hashicorp.com/official-release-channels).
+
+### Installation
+
+To install the beta release of Consul Dataplane, set `VERSION` to `1.0.0-beta` and then follow the instructions to install a specific version of Consul [with the Helm Chart](/docs/k8s/installation/install#install-consul) or [with the Consul-k8s CLI](/docs/k8s/installation/install-cli#install-a-previous-version).
+
+#### Helm
+
+```shell-session
+$ export VERSION=1.0.0-beta
+$ helm install consul hashicorp/consul --set global.name=consul --version ${VERSION} --create-namespace --namespace consul
+```
+
+#### Consul-k8s CLI
+
+```shell-session
+$ export VERSION=1.0.0-beta && \
+    curl --location "https://releases.hashicorp.com/consul-k8s/${VERSION}/consul-k8s_${VERSION}_darwin_amd64.zip" --output consul-k8s-cli.zip
+```
```

Comment:
Would it be possible to move this to index.mdx instead of having this in the the CLI dataplane page?

Reply:
Hi @boruszak I would prefer option 1, since the Consul data plane page looks like more of a CLI reference page as is referenced by the title of consul-data plane.mdx.

I think its fine to include installation material with overview material because that is really how our practitioners ultimately use our docs. This is similar to https://www.consul.io/docs/enterprise/admin-partitions, which personally I think is ok. 

After beta though, we can remove the install instructions since its really the same as what's in our CLI and Helm install but by that time it will be on by default.